packages/atomic/src/components/insight/atomic-insight-interface/e2e/atomic-insight-interface.e2e.ts

```diff
@@ -38,7 +38,7 @@ test.describe('Atomic Insight Panel', () => {
     });
   });
 
-  test.describe('result actions interactions', () => {
+  test.describe.skip('result actions interactions', () => {
```

Comment:
That's not the solution IMO

Reply:
These tests are blocking a bunch of PRs.
I'm fine with skipping them and addressing them in another PR